dynamiq/storages/vector/chroma/chroma.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         connection: Chroma | None = None,
         client: Optional["ClientAPI"] = None,
         index_name: str = "default",
-        create_if_not_exist: bool = True,
+        create_if_not_exist: bool = False,
```

Comment:
I don't know if this the right name? first it's not clear what does it create.
maybe:
- auto_create_index
- create_index_if_absent
- create_index_if_missing
- initialize_index

@acoola, @MaksymAI, @delamainer

Reply:
Generally I see this is the common name and each vector store handles the creation of needed resources:
- Chroma creates collection with proper index
- Pinecone creates index
- Qdrant creates collection + index
- Weaviate creates collection

Reply:
Personally, I don't have strong opinions on naming here. Maybe you right and it's not clear what it creates.

`create_index_if_not_exist`?